test/common/event/dispatcher_impl_test.cc

```diff
@@ -1356,13 +1356,6 @@ class TimerImplTimingTest : public testing::Test {
     while (timer.enabled()) {
       time_system.advanceTimeAndRun(std::chrono::microseconds(1), dispatcher,
                                     Dispatcher::RunType::NonBlock);
-#ifdef WIN32
```

Comment:
Cool, sounds like this is not needed due to the move from Level to EmulatedEdge

Reply:
yes exactly, I validated it with 2k runs locally and there were no flakes